components/__design__/table.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+export const Table: StoryObj = {
+  render: () => (
```

Comment:
Can you add a 2nd example of a headless table? A table without header?

Reply:
Shall I add it in the same file table.stories.tsx or in another new file?

Reply:
tablewithoutheader.stories.tsx @ovflowd

Reply:
Same file, another Story `HeadlessTable` for example.

Reply:
![image](https://github.com/nodejs/nodejs.org/assets/96189881/ce0e0290-1d93-419e-8691-328f68cfef35)
![image](https://github.com/nodejs/nodejs.org/assets/96189881/f78f01dc-312d-4ba1-8a32-69f3b2b26c84)